inc/thorvg.h

```diff
@@ -1390,6 +1390,24 @@ class TVG_EXPORT Saver
      */
     Result save(std::unique_ptr<Paint> paint, const std::string& path) noexcept;
 
+    /**
+     * @brief Export the given @p paint data to the given @p buffer.
+     * If export success, buffer size is returned in @p size.
+     *
+     * @param[in] paint The paint to be saved with all its associated properties.
+     * @param[in] mimeType Mimetype of a data, eg. "tvg".
+     * @param[out] buffer An empty buffer, in which the paint data is to be saved.
+     * @param[out] size Size of a buffer.
+     *
+     * @retval Result::Success When succeed.
+     * @retval Result::Unknown Others.
+     *
+     * @note Saving into a buffer is synchronous. No need to call sync().
```

Comment:
There is no reason to guarantee the sync behavior.
Whatever its behavior does, user could call sync() after save(), it's better for consistency (like other save methods)

Reply:
Do you see some scenario developer may use it asynchronous?
There is no mechanism of callback on finish and I see users (developers) will always call save() and sync() consecutively.
When saving into a file developer may not care further about the saved file so async is great, but for saving into a buffer next step is always passing data somewhere and async is useless if no callback.

Reply:
Yes possibly, user can Save to buffer earlier than use it later.
there is no reason to block the case.